server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -465,6 +465,17 @@
         </wallet>
         <cryptologo cryptocurrency="BCH" file="bch.png"/>
         <cryptologo cryptocurrency="BCH" file="bch.svg"/>
+        <ratesource prefix="telr" name="Telr.io Rate Source">
+            <param name="address" />
+            <param name="secret" />
+            <param name="signature" />
```

Comment:
it looks like address, secret and signature parameters are not being used in the rate source

Reply:
@pvyhnal-generalbytes you are correct. 

those params have not yet been implemented. we have more code to push, but this is just the first of it, _to learn the GB process_. we will be pushing 2 wallets afterwards that will be dependent on this rate source.

no worries, it can be removed now and added later (after implementation) if that is preferred.

Reply:
In that case you will probably need to add it to the tag and not the . Unless it's also needed for the ratesource.

Reply:
@pvyhnal-generalbytes thank you for the update.

> In that case you will probably need to add it to the tag and not the . Unless it's also needed for the ratesource.

what do you mean by, _"add it to the tag and not the"_? 

the params for the rate source are _optional_. only __required__ for rate-limiting. but that shouldn't really be a problem for BATM operators. the reason they are included, is ALSO to be consistent with the __required__ params for the 2 new wallets. 

__option 1:__ i can just remove the params, since the BATMs should never hit the rate limits

__options 2:__ send you a NEW pull with the implemented params

__option 3:__ 1 of 2 wallets (the Cash wallet) is ready to go, tested and in production. if you prefer to see the new ratesource deployed with a working wallet, i can also submit a new pull request w/ wallet.

please advise.

Reply:
sorry, github ate my formatting, I edited my original comment